src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1479,6 +1482,120 @@ class Simplex_tree {
     }
   }
 
+  /** \brief Retrieve good values for extended persistence, and separate the 
+   * diagrams into the ordinary, relative, extended+ and extended- subdiagrams. 
+   * \post This function should be called only if extend_filtration has been called first!
```

Comment:
That looks more like a precondition (`\pre`), something that must be satisfied before the call, than a postcondition, that would be satisfied after the call.

Reply:
Agreed.